.eslintrc

```diff
@@ -12,42 +12,45 @@
         "consistent-return": "off",
         "no-negated-condition": "off",
         "@typescript-eslint/explicit-function-return-type": "error",
-        "@typescript-eslint/array-type": ["error", { "default": "array" }],
+        "@typescript-eslint/array-type": ["error", {"default": "array"}],
         "object-shorthand": "error",
         "no-implicit-globals": "off",
         "camelcase": "off",
         "complexity": "off",
-        "filenames/match-regex": [2, "^[a-z-\\.]+$", true],
+        "filenames/match-regex": [2, "^[a-z-0-9\\.]+$", true],
```

Comment:
Why do we need that?

Reply:
I expanded the rule because I have test case named `grant-s3.test.ts`
https://github.com/gravity-ui/websql-autocomplete/pull/226/files#diff-b066ac93cc472f8ea1ddc67a4299f46ab2e90b2fc58f0fac2f541b15a50c8305